pages.es/common/chsh.md

```diff
@@ -1,20 +1,17 @@
 # chsh
 
-> Cambia el intérprete de comandos de inicio de sesión.
+> Cambia el intérprete de comandos de inicio de sesión del usuario.
```

Comment:
I would add the (login shell) as in 
Cambia el intérprete de comandos de inicio de sesión del usuario (login shell).